Editor/EditorCore/DrawShapeTool.cs

```diff
@@ -108,7 +114,8 @@ void OnEnable()
 
             m_IconContent = new GUIContent()
             {
-                image = IconUtility.GetIcon("Tools/ShapeTool/Arch"),
+                //image = IconUtility.GetIcon("Tools/ShapeTool/Arch"),
+                image = IconUtility.GetIcon("Toolbar/Panel_Shapes"),
```

Comment:
remove unused code

Reply:
this is intentional as tools icons have been reverted to old icons. I wanted to keep the link to the newer icon version when we will update that back later.